src/x-server-local.c

```diff
@@ -391,6 +394,44 @@ got_signal_cb (Process *process, int signum, XServerLocal *server)
     }
 }
 
+static gboolean
+poll_for_socket_cb (XServerLocal *server)
+{
+    XServerLocalPrivate *priv = x_server_local_get_instance_private (server);
+
+    /* Check is X11 socket file exists as an alternative startup test to SIGUSR1 */
+    GStatBuf statbuf;
+    g_autofree gchar *socketpath = g_strdup_printf ("/tmp/.X11-unix/X%d", priv->display_number);
+
+    if ( g_stat (socketpath, &statbuf) == 0 )
```

Comment:
Extra whitespace here doesn't match existing code.

Reply:
I believe you mean blank like 405. I've deleted it. Please let me know if this wasn't it.

Reply:
I mean it should change from `if ( g_stat (socketpath, &statbuf) == 0 )` to `if (g_stat (socketpath, &statbuf) == 0)`